earn/src/components/boost/ImportModal.tsx

```diff
@@ -73,6 +109,46 @@ enum ImportModalState {
   ASKING_USER_TO_MINT,
 }
 
+function getButtonState(state?: ImportModalState) {
+  switch (state) {
+    case ImportModalState.FETCHING_DATA:
+      return {
+        disabled: true,
+        label: 'Loading...',
+      };
```

Comment:
imo these would be more readable on one line, and I'm pretty sure our Prettier config is fine with that